frontend/src/admin/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from "react";
+import DropdownMenuIcon from "../dropdown-menu-icon/dropdown-menu-icon";
+import "./dropdown-menu.scss";
+
+interface DropdownMenuProps{
+    items: string[]
+    onClick: (value: string) => void
+}
+
+export default function DropdownMenu( {items, onClick}: DropdownMenuProps) {
+
+    const [showDropdown, setShowDropdown] = useState<boolean>(false);
+
+    const toggleDropdown = () => {
+        setShowDropdown(!showDropdown);
+    };
+
+    const hideDropdown = (event: React.FocusEvent<HTMLButtonElement>): void => {
+        if (event.currentTarget === event.target) {
+            setShowDropdown(false);
+        }
+    };
+
+    return (
+        <div className={showDropdown ?"c-dropdown-menu--active" : "c-dropdown-menu"}>
```

Comment:
There is minimum missing a space after the '?' for the ternary operator.
However a more readably layout for ternaries are to do the following.

``` 
< Insert Some Boolean Condition >
? < Result in case of true> 
: < Result in case of false>
```
Writing it like this, makes it easier so see what is being returned in case of `true` and `false`.

Reply:
✅ This is changed to follow the more readable layout

Reply:
If the ternary is small enough there is no need for newlining it